health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/utils/utilities.js

```
@@ -360,41 +360,41 @@ function createBoundaryDataByHierarchy(boundaryData) {
-
-      // Build the path from the boundary's parent up to the root
-      while (currentBoundary && currentBoundary.parent) {
-          currentBoundary = boundaryMap[currentBoundary.parent];
-          if (currentBoundary) {
-              path.push(currentBoundary.code);
-          }
+    let path = [];
+    let currentBoundary = boundary;
+
+    // Build the path from the boundary's parent up to the root
+    while (currentBoundary && currentBoundary.parent) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)